amplitude.py

```diff
@@ -161,8 +164,9 @@ def split (payloads, key):
 def send (batches):
     if len(batches["identify"]) > 0:
         # https://amplitude.zendesk.com/hc/en-us/articles/205406617-Identify-API-Modify-User-Properties#request-format
-        requests.post("https://api.amplitude.com/identify",
-                      data={"api_key": AMPLITUDE_API_KEY, "identification": json.dumps(batches["identify"])})
+        response = requests.post("https://api.amplitude.com/identify",
+                                 data={"api_key": AMPLITUDE_API_KEY, "identification": json.dumps(batches["identify"])})
+        response.raise_for_status()
```

Comment:
Need to check response code here.

Reply:
Conversely, does it fail often enough that we need to just eat its failures rather than retrying?

I don't feel super strongly about it either way, I just noticed an apparent bug where it wasn't being checked.  We could put in a comment saying we're ignoring it on purpose :-)

Reply:
> Conversely, does it fail often enough that we need to just eat its failures rather than retrying?

@jbuck, as you're the person who has to check for errors and kick-off retries, it may be best for you to decide this: Would you prefer a failing request to `/identify` be fatal or non-fatal?

Reply:
I almost want to say fatal, but only because we haven't seen much in the way of failure from the Amplitude API at all